### DISCUSSION TITLE
Play this sequence?

### DISCUSSION THREAD

**Discussion by gwhitney** [numberscope/frontscope#313] [Ideas]
The OEIS website allows you to play audio generated from the entries of any sequence (in fact, some have the keyword "hear" suggesting that they sound particularly good). A colleague suggested we should have a "play the audio" button that will produce this or similar audio as the visualizer is drawing. (I think the OEIS actually has pre-computed MIDI files for each sequence that we could just access and use, at least as a first pass on such a feature.)

**Comment by gwhitney** [comment]
Kate also found this reference: http://depts.washington.edu/uwmxl//wordpress/wp-content/uploads/2017/08/noise.pdf for associating sound with sequences, which is I believe a different approach than at oeis,org (and the authors note that the sounds are generally "not melodious"). So potentially there could be a whole category of different "audializers",

**Comment by liammulh** [comment]
http://p5js.org/reference/#/libraries/p5.sound

**Comment by liammulh** [comment]
https://p5js.org/examples/hello-p5-song.html

**Comment by gwhitney** [comment]
This is a fairly speculative proposal, moving to discussions.
